api/controllers/site.js

```diff
@@ -124,10 +125,13 @@ module.exports = {
       return authorizer.removeUser(req.user, site);
     })
     .then(() =>
-      SiteMembershipCreator.revokeSiteMembership({
-        user: req.user,
-        site,
-        userId,
+      SiteMembershipCreator.revokeSiteMembership({ user: req.user, site, userId })
+    )
+    .then(() =>
+      UserActionCreator.addRemoveAction({
```

Comment:
I wonder if in general putting the action creator code into the related service would be a cleaner approach than putting it in the controller code's promise chain. ie, for this one, put the call to `UserActionCreator.addRemoveAction` into `SiteMembershipCreator.revokeSiteMembership`. Thoughts?

Reply:
I don't know if this necessarily needs to be changed now, especially since this is the only use of `UserActionCreator` currently

Reply:
I think you are probably right and it is something to keep in mind...not sure if we should add this change to this PR. I can do a follow-up with the change, though!

Reply:
follow-up works for me